app/presenters/collection_details_presenter_umm_json.rb

```diff
@@ -48,10 +48,16 @@ def initialize(collection, collection_id = nil, token = nil, env = 'prod')
   def doi(doi)
     if doi && doi['DOI']
       doi = doi['DOI']
+      doi = doi.gsub('https://', '')
+      doi = doi.gsub('http://', '')
```

Comment:
- combine two lines to something like ```doi.gsub(/https?:\/\//, '')```
 - Add a test using the collection in the ticket.
Everything else looks good.